plenary/plenary-31.yaml

```diff
@@ -0,0 +1,287 @@
+metadata:
+  title: Resolutions of the 31th plenary meeting of ISO/TC 154
+  date: 2012-10-12
+  source: Secretariat
+resolutions:
+  - dates:
+      - 2012-10-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Publication of ISO 17369"
+    identifier: 006-2012
+    considerations:
+      - type: considering
+        date_effective: 2012-10-12
+        message: considering that no negative votes were received on ISO DIS 17369 (N0690)
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2012-10-12
+        message: resolves to bypass the FDIS ballot and to proceed directly to the publication of ISO 17369.
+
+
+  - dates:
+      - 2012-10-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Resignation of Mr. Michael Dill as convenor of ISO/TC 154/WG 4"
+    identifier: 007-2012
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2012-10-12
+        message: with regret accepts the resignation of its WG 4 convenor Mr. Michael Dill, effective immediately. The TC expresses its appreciation to Mr. Michael Dill for his outstanding contribution to the work of ISO/TC 154. The Technical Committee wishes Mr. Michael Dill every success for the future.
```

Comment:
Here, I believe I haven't put the correct action type. Which one would it be in this case?

Original source:
![capture7](https://user-images.githubusercontent.com/33627611/71759303-95f8f080-2e81-11ea-9b72-cbbeddefca86.JPG)

Reply:
See updates.